ci-operator/config/konflux-ci/e2e-tests/konflux-ci-e2e-tests-main.yaml

```diff
@@ -42,7 +42,7 @@ tests:
     cloud: aws
     labels:
       region: us-east-2
-    owner: rhtap
+    owner: konflux
```

Comment:
I changed only this (optional) job in order to test that cluster pool works correctly. a follow-up PR will be created for switching to konflux cluster pool in all konflux-related jobs